packages/jira-adapter/src/filters/statuses/status_deployment.ts

```diff
@@ -22,6 +22,7 @@ import { deployWithJspEndpoints } from '../../deployment/jsp_deployment'
 import { STATUS_TYPE_NAME } from '../../constants'
 
 const log = logger(module)
+const statusCategoryNameToId = new Map([['TODO', 2], ['DONE', 3], ['IN_PROGRESS', 4]])
```

Comment:
nit: `STATUS_CATEGORY_NAME_TO_ID`
also no need for Map, the native object will work
```suggestion
const STATUS_CATEGORY_NAME_TO_ID: Record<string, number> = {
  TODO: 2,
  ...
}
```